packages/vc-data/src/base/v1.ts

```diff
@@ -15,6 +15,7 @@ export type MonetaryAmountRV1 = CreateThing<
   {
     currency: string
     value: number | string
+    availableValue?: number | string
```

Comment:
Can you add `MonetartyAmountRV1` to the `getBaseV1ContextEntries` function below?